packages/browser-wallet/src/popup/pages/Recovery/RecoveryMain.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useAtom, useAtomValue } from 'jotai';
+import { useTranslation } from 'react-i18next';
+import { networkConfigurationAtom, seedPhraseAtom } from '@popup/store/settings';
+import { popupMessageHandler } from '@popup/shared/message-handler';
+import { InternalMessageType } from '@concordium/browser-wallet-message-hub';
+import { JsonRpcClient, HttpProvider, CryptographicParameters } from '@concordium/web-sdk';
+import { identityProvidersAtom, isRecoveringAtom } from '@popup/store/identity';
+import PendingArrows from '@assets/svg/pending-arrows.svg';
+import { BackgroundResponseStatus } from '@shared/utils/types';
+import { getIdentityProviders } from '@popup/shared/utils/wallet-proxy';
+import { getNet } from '@shared/utils/network-helpers';
+import { NetworkConfiguration } from '@shared/storage/types';
+import PageHeader from '@popup/shared/PageHeader';
+import { absoluteRoutes } from '@popup/constants/routes';
+
+async function getGlobal(network: NetworkConfiguration): Promise<CryptographicParameters> {
```

Comment:
This seems like a function that should be shared, as we have multiple places where we get global.